index.js

```diff
@@ -6,10 +6,12 @@ module.exports = Envie
 Envie.Envie = Envie
 Envie.Joi = Joi
 
-function Envie (description, values, options) {
-  if (!(this instanceof Envie)) return new Envie(description, values, options)
-  if (!values) values = process.env
-  if (!options) options = { noDefaults: false }
+function Envie (description, options = {}) {
+  if (!(this instanceof Envie)) return new Envie(description, options)
+  const {
+    values = process.env,
+    noDefaults = false
```

Comment:
I understand it's confusing if it's the first time you read it - but it's clearer than

```js
const values = typeof options.values === 'undefined' ? options.values : process.env;
const noDefaults = typeof options.noDefaults === 'undefined' ? false : options.noDefaults;
```

.. or to reassign options arg that you can see in a lot of old jQuery-plugins, etc

```js
options = Object.assign({}, { 
  noDefaults: false, 
  values: process.env,
}, options || {});
```

Reply:
I usually do
```javascript
const values = options.values || process.env
```
which I find rather short and self-explanatory. But I mostly find it more used to!

Reply:
This:

```js
const values = options.values || process.env
const noDefaults = options.noDefaults || true
```

Won't work for booleans.. e.g. if we'd implement #8 it'd always be `true`, that's why the `typeof` is needed.

Reply:
If you know how deconstructuring works (been around [since node 6](https://node.green/#ES2015-syntax-destructuring--declarations)) the pattern I do in this PR is self-explanatory as well. Get with the times! :D